src/app/organization/working-days/working-days.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="workingDaysForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <div fxLayout="row" fxLayout.lt-md="column">
+            <h3 fxFlex="40%"> Working Days</h3>
+            <div fxLayout="column" formArrayName="recurrence">
+              <div *ngFor="let day of weekDays; let i = index;">
+                <mat-checkbox [formControlName]="i"> {{day.value}} </mat-checkbox>
+              </div>
+            </div>
+          </div>
+
+          <mat-form-field>
+            <mat-label>Payments due on non working days</mat-label>
+            <mat-select formControlName="repaymentRescheduleType">
+              <mat-option *ngFor="let option of repaymentRescheduleOptionsData;" [value]="option.id">
+                {{option.value}}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+          <div fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="center center">
+            <h3 fxFlex="40%">Extend the term for loans following a daily repayment schedule</h3>
+            <div fxFlex="60%">
+              <mat-checkbox formControlName="extendTermForDailyRepayments"></mat-checkbox>
+            </div>
+          </div>
+
+        </div>
+
+      </mat-card-content>
+
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+        <button mat-raised-button [routerLink]="['../']">Cancel</button>
+        <button mat-raised-button color="primary">Submit</button>
```

Comment:
The button should be disabled until any change is made (as in other edit components of the web app)